packages/experimental/TabList/src/Tab/Tab.types.ts

```diff
@@ -19,9 +19,14 @@ export const tabName = 'Tab';
 
 export interface TabTokens extends FontTokens, IBorderTokens, IForegroundColorTokens, IBackgroundColorTokens, LayoutTokens {
   /**
-   * Horizontal margin of the tab text.
+   * Horizontal left margin of the tab text. If an icon and text are both rendered, this is the margin beween the icon and text.
    */
-  contentMarginHorizontal?: number;
+  contentMarginLeft?: number;
```

Comment:
Should probably be contentMarginStart instead of Left so that it signifies that it adjusts for RTL properly.